core/src/text_instructions.ts

```diff
@@ -125,18 +125,18 @@ function build_instruction_line(
 	var recipe_type = get_recipe(item_name).recipe_type;
 
 	if (recipe_type_functions[recipe_type] === undefined) {
-		return document.createElement("div");
+		return null;
 	}
 
 	return recipe_type_functions[recipe_type](inputs, item_name, generation_totals[item_name], text_item_object);
 }
 
-function build_instruction_inventory_line(
+const build_instruction_inventory_line = (
 	edges: { [key: string]: ResourceEdge },
 	item_name: string
-): HTMLElement | null{
+): HTMLElement | null => {
```

Comment:
Similar note to the above, should we quickly change this from HTMLElement to a span element?